libs/libcups/Makefile

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2006-2012 OpenWrt.org
+# Copyright (C) 2017 Luiz Angelo Daros de Luca <luizluca@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cups
+PKG_VERSION:=1.5.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-source.tar.bz2
+PKG_SOURCE_URL:=http://www.cups.org/software/$(PKG_VERSION)
+PKG_HASH:=8023034e8b58eab1ad7279364e872270a77459f6fb59a84d03229cb1d2a6d0e5
+PKG_MAINTAINER:=Luiz Angelo Daros de Luca <luizluca@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=LICENSE.txt
+
+TARGET_LDFLAGS+=-Wl,-rpath-link=$(STAGING_DIR)/usr/lib
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libcups/Default
+  URL:=http://www.cups.org/
+  SUBMENU:=Printing
+endef
+
+define Package/libcups
+$(call Package/cups/Default)
+  SECTION:=libs
+  CATEGORY:=Libraries
+  DEPENDS:=+zlib +libpthread +libpng +libjpeg
+  TITLE:=Common UNIX Printing System - Core library
+endef
+
+define Package/libcups/description
+	Common UNIX Printing System - Core library
+endef
+
+define Build/Configure
+	$(call Build/Configure/Default, \
+		--with-cups-user="nobody" \
+		--with-cups-group="nogroup" \
+		--with-components="embedded" \
+		--with-pdftops="none" \
+		--without-perl \
+		--without-python \
+		--without-php \
+		--enable-shared \
+		--enable-image \
+		--enable-libusb \
+		--disable-acl \
+		--disable-dbus \
+		--disable-dnssd \
+		--disable-launchd \
+		--disable-ldap \
+		--disable-pam \
+		--disable-slp \
+		--disable-gnutls \
+		--disable-openssl \
+		--disable-cdsassl \
+		--disable-ssl \
+		--disable-gssapi \
+		--disable-tiff, \
```

Comment:
Please add these into CONFIGURE_ARGS variable, then you can remove the custom Build/Configure step